pages/examples/withplatform/geoProcessGraphic.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html> 
+<html>
+    <head>
+        <title>Geo-Process Graphic</title>
+
+        <!-- styles are only used for styling header and auth elements, where possible -->
+        <link rel="stylesheet" type="text/css" href="../styles.css" />
+
+        <!-- boilerplate headers are injected with head.js, grab them from the live example header, or include a link to head.js -->
+        <script src="../boilerplate/head.js"></script>
+
+        <!-- boilerplate auth code is injected with auth.js, check it out for auth setup -->
+        <script src="../boilerplate/auth.js"></script>
+
+        <!-- Add references to the Azure Maps Map control JavaScript and CSS files. -->
+  <link rel="stylesheet" href="https://atlas.microsoft.com/sdk/javascript/mapcontrol/2/atlas.min.css" type="text/css">
```

Comment:
wait so does none of this stuff come in the actual npm package?

Reply:
ooo never noticed. i think i added before finding out it could be installed through node